content/blog/eng/protected-friday.md

```diff
@@ -0,0 +1,126 @@
+---
+authors: ["engineering"]
+comments: true
+date: "2023-04-17"
+draft: false
+share: true
+categories: [Venerdì Protetto]
+title: "Venerdì Protetto"
+languageCode: "en-US"
+type: "post"
+twitterImage: '/images/logo_engineering.png'
+toc: false
+---
+
+# Venerdì Protetto
+
+## What is it all about?
+
+At Facile.it, we strongly believe in sharing knowledge and experiences. Thus, we dedicate one Friday a month to expanding our knowledge and learning among colleagues in the IT department. During these days called Venerdì Protetto (i. e. Protected Friday), we welcome speakers and participate in talks on IT-related topics. Speakers who bring to life Venerdì Protetto are colleagues or external guests.
+
+    
+    
+## Curious to learn more?
+
+Here you can find the talks of each Venerdì Protetto.

```

Comment:
Do you plan to list ALL talks here? It may devolve in a huge list soon!

Wouldn't be better to have one post per month, use a common tag and use this one as a "blueprint"?

Reply:
I agree. That was the original plan, but couldn't find a way to do it. Still not sure if I can pull it off. I did create two separate pages but don't know how to use a common tag. Do you have any further suggestions? Thank you!

Reply:
You already applied the category here: https://github.com/facile-it/facile-it.github.io/pull/58/files#diff-5df0b64bcf65831f9b4859fbc0bc3b0680e95d5eb8827a89ce0c93ff0da773a9R7

Thanks to that, a listing page for that category will be generated, and linked at the bottom of every post that is in it.
Example of a category list: https://engineering.facile.it/categories/conferences/
So you'll have to link `/categories/venerdi-protetto/`

Reply:
Thank you!